benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -34,18 +37,88 @@
 # accdidentaly break our repo.
 OSS_FUZZ_DIR = os.path.join(utils.ROOT_DIR, 'third_party', 'oss-fuzz')
 OSS_FUZZ_REPO_PATH = os.path.join(OSS_FUZZ_DIR, 'infra')
-
 sys.path.append(OSS_FUZZ_REPO_PATH)
```

Comment:
Can you get rid of this sys.path.append and these constants can now be defined after the imports.

Reply:
yes right, @Tanq16  also can start in a new PR, first adding the bisector, git, etc function first with tests in one PR, then this PR of using OSS-Fuzz projects, and then another one migrating projects with OSS-Fuzz files.

Reply:
I removed the zlib extra fuzz targets and modified only the integration python script. Will do PR for all migration separately. Hope that's alright.